exp/controllers/azuremangedcontrolplane_reconciler.go

```diff
@@ -36,6 +36,8 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/cloud/scope"
 	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/groups"
 	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/managedclusters"
+	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/subnets"
+	"sigs.k8s.io/cluster-api-provider-azure/cloud/services/virtualnetworks"
```

Comment:
let's add an e2e test for managed control plane clusters as part of moving it out of exp/ (can be optional on PRs). Is there already an issue tracking that?

Reply:
yeah, I should probably do this sooner rather than later

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/611